consensus/istanbul/types.go

```diff
@@ -277,6 +277,7 @@ type Message struct {
 	Address       common.Address
 	Signature     []byte
 	CommittedSeal []byte
+	DestAddresses []common.Address // This is non nil for consensus messages sent from a proxied validator
```

Comment:
I think it might be cleaner if instead we had:
```
type FwdMessage struct {
  Message Message
  DestAddresses []common.address
}
```

rather than overloading the existing message struct.  Wdyt?

Reply:
Would also allow us to revert `PayloadNoSigAndDestAddrs` to `PayloadNoSig`

Reply:
Ya, that makes sense.  I guess I was trying to figure out how to not have to reimplement `rlp enode/decode`, `Payload`, etc...

Reply:
I created a follow up PR to refactor the istanbul messages a bit:  https://github.com/celo-org/celo-blockchain/pull/583